src/routes/auth.js

```diff
@@ -1,8 +1,11 @@
 import express from 'express';
 import AuthController from '../controllers/authController';
+import schemas from '../validation/schemas';
```

Comment:
Check below comment, if you structure your schema to be that way, you can import the schema with this format.

`
import { registerSchema } from '../validation/schemas';
`